<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSearchWidget</class>
 <widget class="QWidget" name="GlobalSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GlobalSearchWidget</string>
  </property>
  <widget class="QLabel" name="GlobalLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchInputTextEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QListWidget" name="resultList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>561</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
